shared/styles/colors.js

```diff
@@ -1,51 +1,77 @@
 // @flow
+// the _on_white are precomputed colors so we can do less blending on mobile
+import {isAndroid} from '../constants/platform'
+
 const colors = {
   beige: '#f7f1eb',
   black: '#000000',
   black_05: 'rgba(0, 0, 0, 0.05)',
+  black_05_on_white: 'rgb(242,242,242)',
```

Comment:
the _on_white variables assume you're on white , its a precomputed alpha blend

Reply:
good comment